src/negbinfit.jl

```diff
@@ -34,36 +34,47 @@ function mle_for_θ(y::AbstractVector, μ::AbstractVector, wts::AbstractVector;
     θ
 end
 
-function negbin(F, D, args...;
-                initialθ::Real=Inf, maxiter::Integer=30, tol::Real=1.e-6,
-                verbose::Bool=false, kwargs...)
+function negbin(F,
+                D,
+                args...;
+                initialθ::Real=Inf,
+                maxiter::Integer=30,
+                atol::Real=1e-6,
+                rtol::Real=1.e-6,
+                verbose::Bool=false,
+                kwargs...)
     if haskey(kwargs, :maxIter)
-        Base.depwarn("'maxIter' argument is deprecated, use 'maxiter' instead", :fit!)
+        Base.depwarn("'maxIter' argument is deprecated, use 'maxiter' instead", :negbin)
         maxiter = kwargs[:maxIter]
     end
     if haskey(kwargs, :minStepFac)
-        Base.depwarn("'minStepFac' argument is deprecated, use 'minstepfac' instead", :fit!)
+        Base.depwarn("'minStepFac' argument is deprecated, use 'minstepfac' instead", :negbin)
         minstepfac = kwargs[:minStepFac]
     end
     if haskey(kwargs, :convTol)
-        Base.depwarn("'convTol' argument is deprecated, use 'tol' instead", :fit!)
+        Base.depwarn("'convTol' argument is deprecated, use 'tol' instead", :negbin)
         tol = kwargs[:convTol]
     end
     if !issubset(keys(kwargs), (:maxIter, :minStepFac, :convTol))
         throw(ArgumentError("unsupported keyword argument"))
     end
+    if haskey(kwargs, :tol)
+        Base.depwarn("`tol` argument is deprecated, use `atol` and `rtol` instead", :negbin)
+        tol = kwargs[:tol]
```

Comment:
I guess this should be `rtol`? Same above if we want the `convTo` deprecation to work.